metadata-ingestion/src/datahub/api/dataprocess/datajob.py

```diff
@@ -0,0 +1,215 @@
+from dataclasses import dataclass, field
+from typing import (
+    TYPE_CHECKING,
+    Callable,
+    Dict,
+    Iterable,
+    List,
+    Optional,
+    Set,
+    Union,
+    cast,
+)
+
+import datahub.emitter.mce_builder as builder
+from datahub.api.dataprocess.dataflow import DataFlowUrn
+from datahub.api.dataprocess.urn import DataJobUrn
+from datahub.emitter.mcp import MetadataChangeProposalWrapper
+from datahub.metadata.schema_classes import (
+    AuditStampClass,
+    AzkabanJobTypeClass,
+    ChangeTypeClass,
+    DataJobInfoClass,
+    DataJobInputOutputClass,
+    DataJobSnapshotClass,
+    GlobalTagsClass,
+    MetadataChangeEventClass,
+    OwnerClass,
+    OwnershipClass,
+    OwnershipSourceClass,
+    OwnershipSourceTypeClass,
+    OwnershipTypeClass,
+    StatusClass,
+    TagAssociationClass,
+)
+
+if TYPE_CHECKING:
+    from datahub.emitter.kafka_emitter import DatahubKafkaEmitter
+    from datahub.emitter.rest_emitter import DatahubRestEmitter
+
+
+@dataclass
+class DataJob:
```

Comment:
same question here, why is `DataJob` nested under `datahub.api.dataprocess`

Reply:
I agree, this might not the best place and I should move the same place as the model is under `api/datajob`?